js/collections/VerifiedMods.js

```diff
@@ -18,7 +19,7 @@ export default class extends Collection {
   }
 
   url() {
-    return app.localSettings.get('verifiedModsProvider');
+    return app.localSettings.get(`verifiedModsProvider${getCurrentConnection().server.get('useTor') ? 'Tor' : ''}`);
```

Comment:
Is this line too long?

Reply:
Yes it is. I switched editors and didn't notice it had linting turned off and a different line length.